added Testing section

akasma74 opened the pull request:
I saw no such thing in docs and it constantly causes topics like this - https://community.home-assistant.io/t/condition-problems/124222.
Hope this addition is useful.

**Description:**


**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html


<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/9767"><img src="https://gitpod.io/api/apps/github/pbs/github.com/akasma74/home-assistant.io.git/b20b4c5142d11341a0ae10e3ecff3c2fba8afec5.svg" /></a>

akasma74 committed:
added Testing section

I saw no such thing in docs and it constantly causes topics like this - https://community.home-assistant.io/t/condition-problems/124222.
Hope this addition is useful.

klaasnicolaas commented on source/_docs/automation.markdown:
```diff
@@ -60,3 +60,19 @@ automation:
+All this makes that Trigger feature pretty limited and nearly useless for debugging purposes so you need to find another way.
+Make sure you check and adapt to your circumstances appropriate examples from Automation Trigger, Conditions and Actions.
+It is also useful to go to Configuration -> General and click on Check Config button in Configuration validation section to make sure there are no syntax errors before restarting Home Assistant.
+If your automation uses templates in any part, you can do the following to make sure it works as expected:
```
Put the steps in a list with numbers?

klaasnicolaas commented:
![image](https://user-images.githubusercontent.com/20448157/60590483-623af200-9d9c-11e9-8d9d-88e15ecf4cf0.png)

Maybe you should make the text more readable and to the point. Describe how you can test your automations in different ways in a step by step guide?

akasma74 committed:
make it more readable

akasma74 reviewed (commented):
I hope this is more readable than it was

klaasnicolaas commented:
I have thought again about this PR and we think it is better to put your additions on this page: https://www.home-assistant.io/docs/automation/troubleshooting/

akasma74 commented:
I'm happy if the info is somewhere in docs so if you think it's better to be there, that's fine with me.
Probably add link to that page as well?

klaasnicolaas commented:
Are you still moving the text to the page we sugested?

akasma74 committed:
moving Testing section to this better place

as discussed here https://github.com/home-assistant/home-assistant.io/pull/9767#issuecomment-511128560

akasma74 committed:
moving Testing bit to Troubleshooting

as discussed here https://github.com/home-assistant/home-assistant.io/pull/9767#issuecomment-511128560

akasma74 commented:
I've just done that, you can review.

klaasnicolaas committed:
:pencil2: Tweak

klaasnicolaas commented:
I have made some changes to the text, find it quite *long-winded* to read and a reader must know what is meant at the end.

I also see that the troubleshooting page is not listed as an item in the docs menu, so we have to solve that.

akasma74 commented:
Couldn't see anything special so happy with your changes.

frenck committed:
:ambulance: Removing linkable_title

akasma74 reviewed (commented):
good catch ;)
